api/mesh/v1alpha1/mesh_helpers.go

```diff
@@ -0,0 +1,8 @@
+package v1alpha1
+
+func (m *Mesh) IsPassthrough() bool {
+	if m.GetNetworking() == nil || m.GetNetworking().GetOutbound() == nil || m.GetNetworking().GetOutbound().GetPassthrough() == nil {
```

Comment:
nit: you can do just `m.GetNetworking().GetOutbound().GetPassthrough() == nil`, check implementation of those getters, they handle nulls